src/Cones/wsosinterpepinormone.jl

```diff
@@ -320,38 +320,39 @@ function update_inv_hess_prod(cone::WSOSInterpEpiNormOne{T}) where {T}
     U = cone.U
     R = cone.R
     schur = cone.tmpUU2
+    hess_diag_facts = cone.hess_diag_facts
+    Diz = cone.tmpURU2
     schur_backup = cone.tmpUU
+
     @views copyto!(schur, Symmetric(cone.hess_diag_blocks[1], :U))
-    Diz = cone.tmpURU2
 
     @inbounds for r in 2:R
         r1 = r - 1
         diag_r = cone.hess_diags[r1]
-        idxs = block_idxs(U, r)
-        idxs2 = block_idxs(U, r1)
-        LinearAlgebra.copytri!(cone.hess_edge_blocks[r1], 'U')
-        z = cone.hess_edge_blocks[r1]
+
         copyto!(diag_r, cone.hess_diag_blocks[r])
-        cone.hess_diag_facts[r1] = cholesky!(Symmetric(diag_r, :U), check = false)
-        if !isposdef(cone.hess_diag_facts[r1])
+        hess_diag_facts[r1] = cholesky!(Symmetric(diag_r, :U), check = false)
+        if !isposdef(hess_diag_facts[r1])
+            # attempt recovery NOTE can do what hessian factorization fallback does if below is not enough
             copyto!(diag_r, cone.hess_diag_blocks[r])
-            diag_pert = 1 + T(1e-5)
-            for i in 1:U
-                diag_r[i, i] *= diag_pert
-            end
-            cone.hess_diag_facts[r1] = cholesky!(Symmetric(diag_r, :U))
+            increase_diag!(diag_r)
+            hess_diag_facts[r1] = cholesky!(Symmetric(diag_r, :U))
         end
-        @views ldiv!(Diz[idxs2, :], cone.hess_diag_facts[r1], z)
-        @views mul!(schur, z', Diz[idxs2, :], -1, true)
+
+        z = cone.hess_edge_blocks[r1]
+        LinearAlgebra.copytri!(z, 'U')
+        idxs2 = block_idxs(U, r1)
+        @views Dizi = Diz[idxs2, :]
+        ldiv!(Dizi, hess_diag_facts[r1], z)
+        mul!(schur, z', Dizi, -1, true)
     end
 
     copyto!(schur_backup, schur)
     s_fact = cone.hess_schur_fact = cholesky!(Symmetric(schur, :U), check = false)
     if !isposdef(s_fact)
-        for i in 1:U
-            schur_backup[i, i] += 1
```

Comment:
@lkapelevich I assumed this `1` was a mistake, it can change the matrix by a lot